test/test_cli.py

```diff
@@ -47,13 +47,17 @@ def _get_output(testcase, monkeypatch, capsys, **options):
     if directive:
         pytest.skip(f'{directive} directive test')
 
-    options = options.get('directive-options', {})
+    domain = options.get('domain', None)
```

Comment:
Isn't the `None` default superfluous?

Reply:
Hmm, it appears so. For some reason I have the habit of doing it. Maybe in older Python versions it would raise an exception? Anyway, I'll remove them, it's not the only one.